Dockerfile

```diff
@@ -12,19 +12,23 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-FROM debian:stretch
+FROM golang:1.11-stretch
 MAINTAINER Stefan Schimanski <sttts@redhat.com>
 RUN apt-get update \
  && apt-get install -y -qq git=1:2.11.0-3+deb9u4 \
  && apt-get install -y -qq mercurial \
  && apt-get install -y -qq ca-certificates curl wget jq vim tmux bsdmainutils tig gcc \
  && rm -rf /var/lib/apt/lists/*
 
-ENV GOPATH="/go-workspace"
-ENV GOROOT="/go-workspace/go"
-ENV PATH="${GOPATH}/bin:/go-workspace/go/bin:${PATH}"
 ENV GIT_COMMITTER_NAME="Kubernetes Publisher"
 ENV GIT_COMMITTER_EMAIL="k8s-publishing-bot@users.noreply.github.com"
+RUN git config --global user.email "Kubernetes Publisher"
+RUN git config --global user.name "k8s-publishing-bot@users.noreply.github.com"
```

Comment:
Wondering why we used env var. It failed without?

Reply:
i will probably drop this and move it into code